src/main/webapp/app/tutor-course-dashboard/tutor-participation-graph/tutor-participation-graph.component.html

```diff
@@ -0,0 +1,88 @@
+<style type="text/css">
```

Comment:
Move style to own file.
In the component.ts file, you can define a style sheet file in the `@Component` Annotation.
```ts
@Component({
    selector: 'jhi-tutor-participation-graph',
    templateUrl: './tutor-participation-graph.component.html',
    stylesUrls: ['./tutor-participation-graph.component.scss']
})
export class TutorParticipationGraphComponent implements OnInit, OnChanges {}
```

Reply:
Done :)